pkg/kubelet/container/container_gc.go

```diff
@@ -80,5 +82,6 @@ func (cgc *realContainerGC) GarbageCollect() error {
 }
 
 func (cgc *realContainerGC) DeleteAllUnusedContainers() error {
+	glog.Infof("attempting to delete unused containers")
```

Comment:
this the right log level? how often do we expect this will be output?

Reply:
This is essentially just moved from the deleteTerminatedContainers function in the eviction manager I removed.  This keeps the log level the same as before.

Reply:
This will only be output when under disk pressure